index.html

```diff
@@ -19,10 +19,11 @@
             <a class="logo">Femi</a>
```

Comment:
- [ ] Your page is overlapping, kindly fix that.

Reply:
Fixed, thanks.